src/content/content.h

```diff
@@ -50,6 +55,10 @@ struct ContentSpec
 	std::string textdomain;
 };
 
+std::string contentTypeToString(ContentType &t);
```

Comment:
This should be `content_type_to_string`.

Reply:
Look 2 lines below that function. Those were in camelCase as well and I didn't write those. What do you want me to do there?

Reply:
I've rewritten that function's name but left the other 2 alone. If you want me to change those, do tell.

Reply:
Those are methods as opposed to non-method functions and therefore are named in the correct format. I don't know why we name methods and non-methods differently - that's just the way it is in our style guide. 😄

Reply:
They aren't methods, they are just wrapped in a namespace. They are regular functions. They really should be camelCase